service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -88,6 +88,8 @@ if [[ -n "$TERRA_SSH_KEY" ]]; then
   sudo -u "$JUPYTER_USER" sh -c 'ssh-add .ssh/id_rsa; ssh-keyscan -H github.com >> ~/.ssh/known_hosts'
 fi
 
+sudo -u "${JUPYTER_USER}" sh -c "pip install pandas_gbq"
```

Comment:
I don't see any other `pip` commands in this script - just to confirm, do notebooks have `pip` installed already?

Reply:
I just created a notebook, waited for startup script to finish, and ran `pip install pandas_gbq`. It worked.

FYI - default Vertex AI image also has some python packages preinstalled. They have `pandas` preinstalled, but not `pandas_gbq`.